_component_design/tabs.md

```diff
@@ -7,20 +7,128 @@ usage: Teaser Text
 preview-image: components/preview-images/tabs.svg
 resource: true
 status: unknown
-last-modified: 2017-08-17
+last-modified: 2017-09-11
 ---
 
 {% include toc.html %}
 
-Tabs are used to separate content into different views, which can be displayed
-one at a time without a page refresh. Tabs provide easy access to additional
-content without requiring the user to either navigate to a new page or scroll
-too much. The default view should always display the left-most tab.
+Use tabs to separate content into different views that can each be
+displayed quickly without a page refresh. Tabs provide easy access to
+a set of content without requiring the user to either move to a new
+page or scroll across a page.
 
-{% figure [caption:"Default state"] [class:"image bg-light border col-md-12"] %}
-![]({{site.cdn_url}}/img/components/tabs.svg){:width="80%"}
+
+# Introduction
+
+Tabs separate content into different views, making it easy to explore
+content by clicking a descriptive tab title. Switching between tabs is
+quick, since it doesn't require scrolling or refreshing the page. Tabs
+enable content organization at a high level, such as switching between
+views, data sets, or functional aspects of an application.
+
+{% figure [caption:"A generic tab"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/tabs/tabs-lorem-ipsum.svg){:width="80%"}
 {% endfigure %}
 
-{% figure [caption:"Hover state"] [class:"image bg-light border col-md-12"] %}
-![]({{site.cdn_url}}/img/components/tabs-hover.svg){:width="80%"}
+
+## When to use tabs
+
+Use tabs to organize content at a high level. For example, you could
+use tabs to present different sections of a newspaper: one section per
+tab.
+
+Use fixed tabs only with a limited number of tabs and when consistent
+placement would aid muscle memory.
+
+Use scrollable tabs only when there are many tabs or a variable (not
+fixed) number of tabs.
+
+
+## Problems solved by tabs
+
+  * Tabs eliminate the need to scroll through long pages to find the
+    right section. With tabs, it's easy to see what sections are
+    available.
+
+  * Tabs eliminate the need to stack sections.
+
+
+## Best practices for tabs
+
+  * There must be only one active tab per page.
+
+  * The default view must always display the left-most tab.
+
+  * Present tabs as a single row. Wrap tab labels to a second line if
+    needed, and then truncate.
+
+  * Do not include another set of tabbed content within a tab. (No
+    nested tabs.)
+
+  * Highlight the tab corresponding to the visible content.
+
+  * Don't force users to jump back and forth between tabs to perform a
```

Comment:
This seems like a recommendation, rather than a singular instruction like the rest of these dot points.

Reply:
changed